backtester/eventhandlers/portfolio/portfolio.go

```diff
@@ -258,9 +260,9 @@ func (p *Portfolio) OnFill(ev fill.Event, funds funding.IFundReleaser) (fill.Eve
 			}
 		}
 	}
-	err = p.setHoldingsForOffset(&h, true)
+	err = p.SetHoldingsForOffset(&h, true)
 	if errors.Is(err, errNoHoldings) {
-		err = p.setHoldingsForOffset(&h, false)
+		err = p.SetHoldingsForOffset(&h, false)
```

Comment:
One is set to overwrite `true`, but since it doesn't exist, it needs to be set with `false`. While a questionable design, its been in place since the beginning of the backtester. This sounds like another reconsideration outside of this PR